Simplenote/Options.swift

```diff
@@ -111,6 +111,19 @@ extension Options {
         }
     }
 
+    /// Notes List: Sort Mode when Searching!
+    ///
+    var notesSearchSortMode: SortMode {
+        get {
+            let payload = defaults.integer(forKey: .notesSearchSortMode)
+            return SortMode(rawValue: payload) ?? notesListSortMode
```

Comment:
If we don't have `notesSearchSortMode` stored in the user defaults yet, will `payload` be zero and sort mode the first case in the enum? 

I think this may explain this situation that I got right after launching the app in this branch:

<img width="401" alt="Screenshot 2021-01-07 at 15 54 37" src="https://user-images.githubusercontent.com/54751/103908400-c01f6700-5102-11eb-90fa-6f6016fe7c45.png">